pallets/xst/src/lib.rs

```diff
@@ -704,6 +725,16 @@ impl<T: Config> LiquiditySource<T::DEXId, T::AccountId, T::AssetId, Balance, Dis
         }
     }
 
+    /// ## Implementation
+    ///
+    /// The implementation of this method might be quite confusing,
+    /// [here](https://hackmd.io/@jTXlKyDTTYuWUChtsJYcCg/SJ5ItHw12)
+    /// you can find the main logic and formulae without too
+    /// much technical details
+    ///
+    /// Unfortunately, confluence and rustdoc do not render math
```

Comment:
I think we can try to add formulas support to our project. It also would be useful for another pallets docs
https://docs.rs/rustdoc-katex-demo/0.1.5/rustdoc_katex_demo/

Reply:
<img width="762" alt="image" src="https://user-images.githubusercontent.com/8144358/225004827-a241bd28-df57-47d4-92f4-b56dde3f8fc6.png">
Added & checked locally